### PR TITLE
Add Product Marketing role

### PR DESCRIPTION
**Pull request by homeyer**

### CODE REVIEW

File: CAREERS.md [hunk 1]
```diff
@@ -48,5 +48,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+### Product Marketing
+
+We’re hiring a Product Marketing Manager to own Waffle’s positioning in the market, messaging and communication to customers, leads, partners, and the press. We are looking for someone who can deeply understand our customers and our market to help inform and influence our product roadmap in pursuit of continued growth (or product/market fit), while simultaneously building and strengthening our customer community and brand.
+
+This self-starting leader must be a results-oriented strategic thinker with an exceptional blend of multi-audience communication skills (you’ll be talking to roles across software teams, from marketing to engineers), a deep and passionate knowledge of the software development lifecycle and work management landscape, and the ability to plan, manage, and execute strategic initiatives. We are looking for someone who is both metrics and data driven with demonstrated, measured results and a passion for achieving targets.
+
+#### Responsibilities
+
+##### Product Marketing and Positioning
+- Define target segments and buyer and influencer personas for marketing and sales outreach
```
Comment (marydavis) [inline CAREERS.md]: personally not a fan of creating 'personas'

File: CAREERS.md [hunk 2]
```diff
@@ -48,5 +48,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+- Develop market positioning, messaging and marketing campaigns for target segments, including ownership of our social media presence via Twitter, Facebook, and other relevant outlets as needed (eg: Reddit, ProductHunt, Quora) 
+- Iteratively adjust and refine our market positioning and messaging based on customer data and analysis (via customer interviews, A/B testing, metrics, etc) to increase customer acquisition, activation, and retention
+- Perform competitive analysis and the development of competitive selling tools 
+- Perform market, customer, and competitive research to differentiate Waffle’s offerings, as well as to anticipate competitive response
+- Share learning and best practices with the team, continually improve processes and content, and provide clear and consistent reporting to broader stakeholders 
+- Product Life Cycle Communications
```
Comment (marydavis) [inline CAREERS.md]: should this be a link to something?

Comment (homeyer) [inline CAREERS.md]: just gonna remove it, pretty vague

File: CAREERS.md (hunk 1 above)
Comment (homeyer) [inline CAREERS.md]: heh, i'll remove it and i'll let you poke candidates about it :)

**Comment by songawee** [comment]
:+1:

**Comment by marydavis** [comment]
👍

File: CAREERS.md (hunk 1 above)
Comment (marydavis) [inline CAREERS.md]: haha, thanks :)

**Pull request merged**